rust/worker/src/execution/operators/merge_knn_results.rs

```diff
@@ -109,16 +110,30 @@ impl Operator<MergeKnnResultsOperatorInput, MergeKnnResultsOperatorOutput>
                             }
                         }
                     }
-                    merge_results(
-                        &hnsw_result_user_ids,
-                        &input.hnsw_result_distances,
-                        &hnsw_result_vectors,
-                        &input.brute_force_result_user_ids,
-                        &input.brute_force_result_distances,
-                        &input.brute_force_result_vectors,
-                        input.include_vectors,
-                        input.k,
-                    )
+
```

Comment:
nit: can we cleanup the println! in this file while here - either by removing them or turning into traces